doc/superbuild/source/developers/python/index.rst

```diff
@@ -0,0 +1,246 @@
+..
+   Author(s): Cody J. Balos @ LLNL
+   -----------------------------------------------------------------------------
+   SUNDIALS Copyright Start
+   Copyright (c) 2002-2025, Lawrence Livermore National Security
+   and Southern Methodist University.
+   All rights reserved.
+
+   See the top-level LICENSE and NOTICE files for details.
+
+   SPDX-License-Identifier: BSD-3-Clause
+   SUNDIALS Copyright End
+   -----------------------------------------------------------------------------
+
+.. __Developer.Python:
+
+Python Interfaces
+=================
+
+This chapter covers details developers need to know about the SUNDIALS Python interfaces, distributed as the Python package sundials4py.
+
+We use `nanobind <https://github.com/wjakob/nanobind>__` for the Python bindings. nanobind is a sleeker, faster ``pybind11``.
```

Comment:
Is nanobind a fork/version of pybind11?  Is it just a completely separate package?  As written, it sounds like the developers of pybind11 just cleaned things up and changed the name.
```suggestion
We use `nanobind <https://github.com/wjakob/nanobind>__` for the Python bindings. nanobind is a sleeker, faster replacement for ``pybind11``.
```